deployment/kustomizations/base/deploy-wren-engine.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wren-engine-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: wren-engine
+  template:
+    metadata:
+      labels:
+        app: wren-engine
+    spec:
+      volumes:
+        - name: wren-data
+          persistentVolumeClaim:
+            claimName: wren-data-pvc
+      initContainers:
+        - name: bootstrap
+          image: ghcr.io/canner/wren-bootstrap:0.1.4
+          env:
+            - name: DATA_PATH
+              valueFrom:
+                configMapKeyRef:
+                  name: wren-config
+                  key: DATA_PATH
+            # PG_HOST is not implemented yet
+            # Do we actually need any PG_* variable for the bootstrap/init?
```

Comment:
Well, we do need this for now. I think we're going to deprecate `PG_USERNANE`, `PG_PASSWORD` variables in the future releases.

Reply:
Okay, so it'll stays for now.